practice/1_python_part_1/task6.py

```diff
@@ -20,5 +20,13 @@
 
 
 def get_min_max(filename: str) -> Tuple[int, int]:
-    ...
-
+  l = []
+  with open(filename) as opened_file:
+    for line in opened_file:
+      l.append(line.replace('\n',''))
+  for digit in l:
+    if digit.isdigit() or digit.startswith('-') and digit [1:].isdigit() == True:
+      l[l.index(digit)] = int(l[l.index(digit)])
```

Comment:
Why do need to get element by index in this row? Can it be done by using `digit` itself?
```
l.append(int(digit))
```

Additional question :slightly_smiling_face:: Find out a performance of list#index operation